.github/workflows/bandit.yml

```diff
@@ -1,7 +1,10 @@
 # Bandit static analysis (for Python code)
 name: Bandit
-
-on: [push, pull_request]
+on:
+  push:
+    branches-ignore:
+      - 'dependabot/**'
+    pull_request:
```

Comment:
PR should be aligned with push, not with "branches-..." (pls apply everywhere)

Reply:
done